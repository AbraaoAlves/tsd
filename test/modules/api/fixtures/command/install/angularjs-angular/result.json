{
  "nameMatches": [
    {
      "path": "angularjs/angular.d.ts",
      "project": "angularjs",
      "name": "angular",
      "def": {
        "path": "angularjs/angular.d.ts",
        "commit": {
          "commitSha": "a8d8c2ac668fbb99cb60fb5d25c663e9356fee14"
        },
        "blob": {
          "sha": "c44f3408e4b90c2572045d231a91f6042cf2e8da"
        },
        "key": "angularjs/angular.d.ts-a8d8c2ac668fbb99cb60fb5d25c663e9356fee14",
        "solved": true,
        "dependencies": [
          {
            "path": "jquery/jquery.d.ts",
            "project": "jquery",
            "name": "jquery",
            "def": {
              "path": "jquery/jquery.d.ts",
              "commit": {
                "commitSha": "a8d8c2ac668fbb99cb60fb5d25c663e9356fee14"
              },
              "blob": {
                "sha": "99969f710e2dbcac67382988edc7224550d29b86"
              },
              "key": "jquery/jquery.d.ts-a8d8c2ac668fbb99cb60fb5d25c663e9356fee14",
              "solved": true,
              "dependencies": []
            },
            "history": []
          }
        ]
      },
      "history": []
    }
  ],
  "selection": [
    {
      "path": "angularjs/angular.d.ts",
      "commit": {
        "commitSha": "a8d8c2ac668fbb99cb60fb5d25c663e9356fee14"
      },
      "blob": {
        "sha": "c44f3408e4b90c2572045d231a91f6042cf2e8da"
      },
      "key": "angularjs/angular.d.ts-a8d8c2ac668fbb99cb60fb5d25c663e9356fee14",
      "solved": true,
      "dependencies": [
        {
          "path": "jquery/jquery.d.ts",
          "project": "jquery",
          "name": "jquery",
          "def": {
            "path": "jquery/jquery.d.ts",
            "commit": {
              "commitSha": "a8d8c2ac668fbb99cb60fb5d25c663e9356fee14"
            },
            "blob": {
              "sha": "99969f710e2dbcac67382988edc7224550d29b86"
            },
            "key": "jquery/jquery.d.ts-a8d8c2ac668fbb99cb60fb5d25c663e9356fee14",
            "solved": true,
            "dependencies": []
          },
          "history": []
        }
      ]
    }
  ],
  "written": {
    "D:\\_Editing\\github\\tsd\\tsd-origin\\test\\modules\\api\\tmp\\command\\install\\angularjs-angular\\typings\\jquery\\jquery.d.ts": {
      "path": "jquery/jquery.d.ts",
      "commit": {
        "commitSha": "a8d8c2ac668fbb99cb60fb5d25c663e9356fee14"
      },
      "blob": {
        "sha": "99969f710e2dbcac67382988edc7224550d29b86"
      },
      "key": "jquery/jquery.d.ts-a8d8c2ac668fbb99cb60fb5d25c663e9356fee14",
      "solved": true,
      "dependencies": []
    },
    "D:\\_Editing\\github\\tsd\\tsd-origin\\test\\modules\\api\\tmp\\command\\install\\angularjs-angular\\typings\\angularjs\\angular.d.ts": {
      "path": "angularjs/angular.d.ts",
      "commit": {
        "commitSha": "a8d8c2ac668fbb99cb60fb5d25c663e9356fee14"
      },
      "blob": {
        "sha": "c44f3408e4b90c2572045d231a91f6042cf2e8da"
      },
      "key": "angularjs/angular.d.ts-a8d8c2ac668fbb99cb60fb5d25c663e9356fee14",
      "solved": true,
      "dependencies": [
        {
          "path": "jquery/jquery.d.ts",
          "project": "jquery",
          "name": "jquery",
          "def": {
            "path": "jquery/jquery.d.ts",
            "commit": {
              "commitSha": "a8d8c2ac668fbb99cb60fb5d25c663e9356fee14"
            },
            "blob": {
              "sha": "99969f710e2dbcac67382988edc7224550d29b86"
            },
            "key": "jquery/jquery.d.ts-a8d8c2ac668fbb99cb60fb5d25c663e9356fee14",
            "solved": true,
            "dependencies": []
          },
          "history": []
        }
      ]
    }
  }
}