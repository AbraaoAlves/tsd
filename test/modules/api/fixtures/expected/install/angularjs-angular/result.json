{
  "nameMatches": [
    {
      "path": "angularjs/angular.d.ts",
      "project": "angularjs",
      "name": "angular",
      "def": {
        "path": "angularjs/angular.d.ts",
        "commit": {
          "commitSha": "3bc329f49a971b3dae92ca682cb81816dbe31a53"
        },
        "blob": {
          "sha": "c44f3408e4b90c2572045d231a91f6042cf2e8da"
        },
        "key": "angularjs/angular.d.ts-3bc329f49a971b3dae92ca682cb81816dbe31a53",
        "solved": true,
        "dependencies": [
          {
            "path": "jquery/jquery.d.ts",
            "project": "jquery",
            "name": "jquery",
            "def": {
              "path": "jquery/jquery.d.ts",
              "commit": {
                "commitSha": "3bc329f49a971b3dae92ca682cb81816dbe31a53"
              },
              "blob": {
                "sha": "99969f710e2dbcac67382988edc7224550d29b86"
              },
              "key": "jquery/jquery.d.ts-3bc329f49a971b3dae92ca682cb81816dbe31a53",
              "solved": true,
              "dependencies": []
            },
            "history": []
          }
        ]
      },
      "history": []
    }
  ],
  "selection": [
    {
      "path": "angularjs/angular.d.ts",
      "commit": {
        "commitSha": "3bc329f49a971b3dae92ca682cb81816dbe31a53"
      },
      "blob": {
        "sha": "c44f3408e4b90c2572045d231a91f6042cf2e8da"
      },
      "key": "angularjs/angular.d.ts-3bc329f49a971b3dae92ca682cb81816dbe31a53",
      "solved": true,
      "dependencies": [
        {
          "path": "jquery/jquery.d.ts",
          "project": "jquery",
          "name": "jquery",
          "def": {
            "path": "jquery/jquery.d.ts",
            "commit": {
              "commitSha": "3bc329f49a971b3dae92ca682cb81816dbe31a53"
            },
            "blob": {
              "sha": "99969f710e2dbcac67382988edc7224550d29b86"
            },
            "key": "jquery/jquery.d.ts-3bc329f49a971b3dae92ca682cb81816dbe31a53",
            "solved": true,
            "dependencies": []
          },
          "history": []
        }
      ]
    }
  ],
  "written": {
    "D:\\_Editing\\github\\tsd\\tsd-origin\\test\\modules\\api\\tmp\\result\\install\\angularjs-angular\\typings\\angularjs\\angular.d.ts": {
      "path": "angularjs/angular.d.ts",
      "commit": {
        "commitSha": "3bc329f49a971b3dae92ca682cb81816dbe31a53"
      },
      "blob": {
        "sha": "c44f3408e4b90c2572045d231a91f6042cf2e8da"
      },
      "key": "angularjs/angular.d.ts-3bc329f49a971b3dae92ca682cb81816dbe31a53",
      "solved": true,
      "dependencies": [
        {
          "path": "jquery/jquery.d.ts",
          "project": "jquery",
          "name": "jquery",
          "def": {
            "path": "jquery/jquery.d.ts",
            "commit": {
              "commitSha": "3bc329f49a971b3dae92ca682cb81816dbe31a53"
            },
            "blob": {
              "sha": "99969f710e2dbcac67382988edc7224550d29b86"
            },
            "key": "jquery/jquery.d.ts-3bc329f49a971b3dae92ca682cb81816dbe31a53",
            "solved": true,
            "dependencies": []
          },
          "history": []
        }
      ]
    },
    "D:\\_Editing\\github\\tsd\\tsd-origin\\test\\modules\\api\\tmp\\result\\install\\angularjs-angular\\typings\\jquery\\jquery.d.ts": {
      "path": "jquery/jquery.d.ts",
      "commit": {
        "commitSha": "3bc329f49a971b3dae92ca682cb81816dbe31a53"
      },
      "blob": {
        "sha": "99969f710e2dbcac67382988edc7224550d29b86"
      },
      "key": "jquery/jquery.d.ts-3bc329f49a971b3dae92ca682cb81816dbe31a53",
      "solved": true,
      "dependencies": []
    }
  }
}